_posts/2020-11-19-Setting-Up-a-Pull-Request.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Setting Up a Pull Request
+description: 
+summary: 
+tags: 
+---
+Kajari instructed Celeste and I to start creating a new post using pull request. Pull requests are changes committed in the branch that are prresented to viewers of the users repository on Github. To initiate a new post/pull request, the following commands must be used to complete the task.
+* git checkout -b new post - the purpose of checkout -b is to create a new branch with the name new_post
+[This is the post of reference of other git commands after creating a new post](https://osvaldo178.github.io/2020/09/14/setting-up-git)
+
+After completing the necessary commands and steps, the user and other applicant assigned to the repository can view the new post created, and that applicant added in the repository can view and request changes in the post. After completing those changes, you can merge the branch and complete the post. 
```

Comment:
Add in a screenshot of the yellow banner that shows up when you push up a new branch and a screenshot of the screen that allows you to create the Pull Request. 

Also take those screenshots and put them in the assets folder as we have done in the past. We will fix the issues with getting the images to show next time.